notebooks/lesson01_intro-to-pandas.ipynb

```diff
@@ -0,0 +1,1487 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lesson 1 - Introduction to pandas and seaborn\n",
+    "\n",
+    "> How to use the pandas and seaborn libraries to quickly load data into Python, calculate descriptive statistics, and make simple visualisations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/lewtun/dslectures/master?filepath=notebooks%2Flesson01.ipynb)"
```

Comment:
The binder badge seems to point to the old lesson_01 notebook not the lesson_01_intro.. notebook.

Reply:
Good catch!